src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -15,6 +15,18 @@
   </module>
 </module>
 */
-
+package com.puppycrawl.tools.checkstyle.filters.suppressionsinglefilter;
 // xdoc section -- start
+public class Example1 {
+  // Violation, 'MagicNumberCheck (without suppression)'
```

Comment:
`// violation .....`
We use lowercase in all other places, let's be consistent.
And there should be violation message in single quotes.

Update all other lines in your PR